modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/GridCacheOffheapManager.java

```diff
@@ -434,6 +434,9 @@ private void saveStoreMetadata(
 
                         if (part != null && part.state() != GridDhtPartitionState.EVICTED)
                             state = part.state();
```

Comment:
Why do you not write right here?
`state = part.state() == LOST ? OWNING : part.state()`

Reply:
Done